B9PartSwitch/PartSwitch/PartModifiers/ModuleFuelTanksHandler.cs

```diff
@@ -0,0 +1,47 @@
+
+namespace B9PartSwitch.PartSwitch.PartModifiers
+{
+    public class ModuleFuelTanksHandler : PartModifierBase, IPartAspectLock
+    {
+        public const string PART_ASPECT_LOCK = "ModuleFuelTanks";
+
+        private readonly PartModule module;
+        private readonly ConfigNode originalNode;
+        private readonly ConfigNode dataNode;
+        private readonly BaseEventDetails moduleDataChangedEventDetails;
+        public ModuleFuelTanksHandler(PartModule module, ConfigNode originalNode, ConfigNode dataNode, BaseEventDetails moduleDataChangedEventDetails)
+        {
+            this.module = module;
+            this.originalNode = originalNode;
+            this.dataNode = dataNode;
+            this.moduleDataChangedEventDetails = moduleDataChangedEventDetails;
+        }
+
+        public object PartAspectLock => PART_ASPECT_LOCK;
+        public override string Description => "blah, FIXME";
+        public override void DeactivateOnStartEditor() => Deactivate();
+        public override void ActivateOnStartEditor() => Activate();
+        public override void DeactivateOnSwitchEditor() => Deactivate();
+        public override void ActivateOnSwitchEditor() => Activate();
+        public override void OnWillBeCopiedInactiveSubtype() => Activate();
+        public override void OnWasCopiedInactiveSubtype() => Deactivate();
+        public override void OnWasCopiedActiveSubtype() => Activate();
+        public override void OnBeforeReinitializeInactiveSubtype() => Activate();
+
+        private void Activate() => ApplyNode(dataNode);
+        private void Deactivate() => ApplyNode(originalNode);
+
+        private void ApplyNode(ConfigNode sourceNode) {
```

Comment:
Maybe this works as an initial pass, are we sure we want to use nothing else from the data node though?

Reply:
@drveyl or @nathankell might know of something else that we would want, but pretty sure no.

Reply:
Doesn't seem like we want anything else from discussions with Veyl on Discord